### PR TITLE
Link to MapLibre's code of conduct

### PR DESCRIPTION
**Pull request by wipfli**
## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [ ] Manually test the debug page.
 - [ ] Suggest a changelog category: bug/feature/docs/etc. or "skip changelog".
 - [ ] Add an entry inside this element for inclusion in the `maplibre-gl-js` changelog: `<changelog></changelog>`.

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
 # Code of conduct
 
-A formal code of conduct may be selected by the MapLibre project shortly, until then:
-
-Be kind, thoughtful and respectul to everyone. Give other people the benefit of the doubt.
+The MapLibre code of conduct can be found at https://github.com/maplibre/maplibre/blob/main/CODE_OF_CONDUCT.md
```
Comment (nyurik) [inline CODE-OF-CONDUCT.md:3]: I think the whole content of this file can be just this:

```
[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/maplibre/maplibre/blob/main/CODE_OF_CONDUCT.md)
```

that looks like [Contributor Covenant](https://github.com/maplibre/maplibre/blob/main/CODE_OF_CONDUCT.md)

Comment (wipfli) [reply CODE-OF-CONDUCT.md:3]: Yes, but then the email is missing...

Comment (nyurik) [reply CODE-OF-CONDUCT.md:3]: do you want some extra text with email in every repo?  Clicking that button takes users to the shared one that has our email

Comment (wipfli) [reply CODE-OF-CONDUCT.md:3]: Ah, then I just misunderstood you... I like the idea of the button

Comment (wipfli) [inline CODE-OF-CONDUCT.md:3]:
```suggestion
[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/maplibre/maplibre/blob/main/CODE_OF_CONDUCT.md)

```

**Comment by wipfli** [comment]
@nyurik updated...

**Review by nyurik — APPROVED** [review]
thx!

**Pull request merged**